.github/workflows/build-test-image.yaml

```
@@ -614,6 +614,67 @@ jobs:
+            extraParams: "--resource-key 'single-Zone' --instance-name 'test-replication-network-change' --tls --instance-description 'test-replication-network-change' --instance-type 't2.medium' --storage-size '30' --rdb-config 'medium' --aof-config 'always'"
+          - name: PRO/ClusterSingleZone - AWS/us-east-2 - Network change with TLS
+            if: "true"
+            testFile: test_network_change.py
+            tierName: pro-${{ contains(github.ref, 'refs/tags/v') && 'main' || github.event_name == 'pull_request' && github.event.pull_request.head.ref || github.ref_name }}
+            cloudProvider: aws
+            cloudRegion: us-east-2
+            subscriptionId: sub-GJPV3NoNC0
+            serviceId: ${{ vars.OMNISTRATE_INTERNAL_SERVICE_ID }}
+            environmentId: ${{ vars.OMNISTRATE_INTERNAL_DEV_ENVIRONMENT}}
+            extraParams: "--resource-key 'cluster-Single-Zone' --instance-name 'test-cluster-network-change' --tls --instance-description 'test-cluster-network-change' --instance-type 't2.medium' --storage-size '30' --rdb-config 'medium' --aof-config 'always' --cluster-replicas '1' --host-count '6'"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)